src/config.py

```diff
@@ -248,7 +254,9 @@ def __init__(self):
         elif environment == 'Deepin':
             os.system(f"cp -au ./deepin {home}/.config/")
             os.system(f"cp -au ./deepin-data {home}/.local/share/deepin/")
-        elif environment == None:
+        elif environment == 'Hyprland':
+            os.system(f"cp -aur ./hypr {home}/.config/ -v")
+	elif environment == None:
```

Comment:
This row is not aligned correctly, which generates a TabError.

Reply:
done